gen/safeq

```diff
@@ -106,22 +101,22 @@ foreach my $login (sort keys %$usersByLogin) {
 	print FILE "lastname:", checkBase64($attributes->{$A_USER_LAST_NAME} || '(N/A)'), "\n";
 	print FILE "costcenter:", checkBase64("costcenterno=" . ($attributes->{$A_USER_WORKPLACE_ID} || 100) . ", ou=costcenters, o=ysoftsafeq"), "\n";
 	print FILE "email:", checkBase64($attributes->{$A_USER_MAIL}), "\n";
-	print FILE "homedir:: ", encode_base64('\\\\ha-ntc.ics.muni.cz\\profiles\\' . $attributes->{$A_USER_LOGIN});
+	print FILE "homedir:: ", encode_base64('\\\\ha-ntc.ics.muni.cz\\profiles\\' . $login);
 
-	foreach my $chipNumber (@{$attributes->{$A_USER_CHIP_NUM}}) {
+	foreach my $chipNumber (sort @{$attributes->{$A_USER_CHIP_NUM}}) {
```

Comment:
Sorting of chip numbers fails if there is an undef value present (user has not chips). Please add check for undef and skip printing this attribute if its empty. Thank you.